src/Symfony/Component/DependencyInjection/ParameterBag/EnvPlaceholderParameterBag.php

```diff
@@ -94,9 +94,9 @@ public function resolve()
             if (!isset($this->parameters[$name = strtolower("env($env)")])) {
                 continue;
             }
-            if (is_numeric($default = $this->parameters[$name])) {
+            if (is_scalar($default = $this->parameters[$name])) {
```

Comment:
:-1: for changing this class, we should not cast to string. There is no reason to do so.